model/remote_cluster.go

```diff
@@ -11,16 +11,24 @@ import (
 	"encoding/json"
 	"io"
 	"net/http"
+	"regexp"
 	"strings"
 )
 
 const (
 	RemoteOfflineAfterMillis = 1000 * 60 * 5 // 5 minutes
+	RemoteNameMinLength      = 1
+	RemoteNameMaxLength      = 64
```

Comment:
Is this a storage limitation? or a presentation limitation?

Reply:
Storage.  Same as `model.User.Username`